eventheader_macros/src/event_generator.rs

```diff
@@ -239,7 +239,7 @@ impl EventGenerator {
         const _EH_KEYWORD = keywords...;
         const _EH_TAGn: u16 = TAGn;
         static _EH_TRACEPOINT = EventHeaderTracepoint::new(...);
-        static _EH_TRACEPOINT_PTR = &_EH_TRACEPOINT;
+        static mut _EH_TRACEPOINT_PTR = &_EH_TRACEPOINT;
```

Comment:
This comment change doesn't correspond to any code change. (Though I don't think this comment line corresponds to any code, so this comment line should probably be deleted.)

Reply:
It is fixing the comment to match the code and the comment on line 168.